app/controllers/gardens_controller.rb

```diff
@@ -39,7 +39,10 @@ def edit
 
   def create
     @garden.owner_id = current_member.id
-    flash[:notice] = I18n.t('gardens.created') if @garden.save
+    if @garden.save
+      link = new_activity_path(name: 'Weed the garden bed', garden_id: @garden.id, due_date: 2.weeks.from_now.to_date)
+      flash[:notice] = t('gardens.created_prompt_html', link: link).html_safe
```

Comment:
Refactor this to use link_to helpers so html_safe is not required, and repeat for all similar areas.